src/main/resources/templates/courseFormView.twig

```diff
@@ -34,11 +36,58 @@
                     aria-expanded="false" aria-controls="courseForm">Cancel
             </button>
             <input type="submit" class="btn btn-primary" value="Submit" class="form-control">
+            {% if (course.id) %}
+                <div style="float:right">
+                    <button id="deleteButton" type="button" disabled="true" class="btn btn-blue-grey" >
+                        Delete
+                    </button>
+                    <div style="padding: 15px">
```

Comment:
Could we possibly remove the padding on this element, making it padding-bottom instead? I find it looks a bit weird with the checkbox so far away from the button, and with this not aligned properly. Removing the `margin-bottom: 1.5rem;` on the delete button also helps keep things nice. Heres what it would look like:

![image](https://user-images.githubusercontent.com/22732449/32680165-ce4cc4c2-c63f-11e7-8c4c-9101ce4a792a.png)